ios/LayoutReanimation/REAAnimationsManager.h

```diff
@@ -39,6 +46,17 @@ typedef void (^REAAnimationRemovingBlock)(NSNumber *_Nonnull tag);
                        atIndices:(NSArray<NSNumber *> *)indices;
 - (void)onViewCreate:(UIView *)view after:(REASnapshot *)after;
 - (void)onViewUpdate:(UIView *)view before:(REASnapshot *)before after:(REASnapshot *)after;
+- (void)viewsDidLayout;
+- (NSDictionary *)prepareDataForLayoutAnimatingWorklet:(NSMutableDictionary *)currentValues
+                                          targetValues:(NSMutableDictionary *)targetValues;
+- (UIView *)viewForTag:(NSNumber *)tag;
+- (BOOL)hasAnimationForTag:(NSNumber *)tag type:(NSString *)type;
+- (void)clearAnimationConfigForTag:(NSNumber *)tag;
+- (void)startAnimationForTag:(NSNumber *)tag
+                        type:(NSString *)type
+                  yogaValues:(NSDictionary *)yogaValues
+                       depth:(NSNumber *)depth;
+- (void)visitTree:(UIView *)view block:(REATreeVisitor)block;
```

Comment:
Could we rename this to something like `visitTree:withBlock` or `visitTree:withVisitor:` to better match the ObjC style?

Reply:
I wonder if we could use `REANodeFind` instead of this? how is this one different?

Reply:
Under the hood, visit tree is just a `REANodeFind`
```
- (void)visitTree:(UIView *)view block:(REATreeVisitor)block
{
  REANodeFind(view, block);
}
```
But `REANodeFind` is defined as `static BOOL REANodeFind` in the top level of `REAAnimationManager.m` and it wasn't available from other files. I can rewrite `REANodeFind` as a static method in `REAAnimationManager` class if you want

Reply:
Resolved in: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/dd7bac67f17f0b015ef599a2307bdbccf9715713